modules/recording/recording/libs/download.ts

```diff
@@ -0,0 +1,98 @@
+import { PDFDocument, rgb, StandardFonts } from "pdf-lib";
+import { saveAs } from "file-saver";
+
+class Download {
+
+  static async downloadAudio(audioUrl: string) {
+    try {
+      const response = await fetch(audioUrl);
+      const blob = await response.blob();
+      const blobUrl = URL.createObjectURL(blob);
+
+      const anchor = document.createElement("a");
+      anchor.href = blobUrl;
+      anchor.download = `audio-${new Date().toLocaleDateString()}.mp3`;
+
+      document.body.appendChild(anchor);
+      anchor.click();
+      document.body.removeChild(anchor);
+
+      URL.revokeObjectURL(blobUrl);
+    } catch (error) {
+      console.error("Failed to download audio file:", error);
+      throw error;
+    }
+  }
+  
+  static async downloadTranscription(content: string[]) {
+    const pdfDoc = await PDFDocument.create();
+    const page = pdfDoc.addPage([600, 800]);
+    const { height } = page.getSize();
+
+    let y = height - 50;
+    const fontSize = 12;
+    const lineHeight = fontSize * 1.5;
+
+    for (const line of content) {
+      if (y < 50) {
+        // Correctly add a new page to the pdfDoc
+        pdfDoc.addPage();
+        y = height - 50;
+      }
+      page.drawText(line, { x: 50, y: y, size: fontSize });
+      y -= lineHeight;
+    }
+
+    const pdfBytes = await pdfDoc.save();
+    saveAs(
+      new Blob([pdfBytes], { type: "application/pdf" }),
+      "transcription.pdf",
```

Comment:
Why not to calle it `downloadPDF` and pass the name as a prop? Can be used in more scenarios